packages/serverpod/lib/src/generated/database/enum_serialization.dart

```diff
@@ -0,0 +1,28 @@
+/* AUTOMATICALLY GENERATED CODE DO NOT MODIFY */
+/*   To generate run: "serverpod generate"    */
+
+// ignore_for_file: library_private_types_in_public_api
+// ignore_for_file: public_member_api_docs
+// ignore_for_file: implementation_imports
+
+// ignore_for_file: no_leading_underscores_for_library_prefixes
+import 'package:serverpod/serverpod.dart' as _i1;
+
+enum EnumSerialization with _i1.SerializableEntity {
+  byName,
+  byIndex;
+
+  static EnumSerialization? fromJson(int index) {
+    switch (index) {
+      case 0:
```

Comment:
Looks like you didn't run `generate_all` on this yet? This generated file doesn't include the changes in enum class generation in this PR...

Reply:
Indeed seems like I missed that will do that now!